docs/modules/architecture/partials/nav.adoc

```diff
@@ -6,6 +6,7 @@
 ** xref:architecture:event-time-processing.adoc[]
 ** xref:architecture:sliding-window.adoc[]
 ** xref:architecture:in-memory-storage.adoc[]
+** xref:architecture:vector-search-overview.adoc[]
```

Comment:
@shultseva I've given this some thought; I don't think this should be hidden under the Reference section - although under Architecture is the logical place to have such information.

For this release, I'd suggest moving it under the Vector Collection topic from https://github.com/hazelcast/hz-docs/pull/1124 to keep the information together for visibility.

There is work on-going to improve the side navigation, so it might be restructured as part of that; I've also suggested some x-refs between these sections in both this and the other PR just to keep them tied together wherever they end up.

Reply:
so should the navigation look something like that in 5.5:

```
Distributed Data Structures
   ...
  IMap
    Oveview
    ...
  Vector Collection
    Vector Collection (?)  (from https://github.com/hazelcast/hz-docs/pull/1124)
    VectorCollection data structure design   (from this PR)
  JCache
  ...
```
?

Should we introduce intermediate grouping node for 2 vector documentation pages?

Reply:
Do you mean such changes?
801c82f932331d3eef7b8e71d85a18bd1c38a1fb

Reply:
Yes, that's it.